.circleci/config.yml

```diff
@@ -196,6 +177,68 @@ jobs:
           path: build/test-results
       - store_artifacts:
           path: build/reports/tests
+  cas1_e2e_tests:
+    docker:
+      - image: mcr.microsoft.com/playwright:v1.34.1-focal
+    circleci_ip_ranges: true # opt-in to jobs running on a restricted set of IPs
+    steps:
+      - run:
+          name: Clone E2E repo
+          command: |
+            git clone https://github.com/ministryofjustice/hmpps-approved-premises-e2e.git .
+      - run:
+          name: Update npm
+          command: 'npm install -g npm@latest'
+      - node/install-packages
+      - run:
+          name: E2E Check
+          command: |
+            npm run test
+      - store_artifacts:
+          path: playwright-report
+          destination: playwright-report
+      - store_artifacts:
+          path: test-results
+          destination: test-results
+      - slack/notify:
+          event: fail
+          channel: << pipeline.parameters.alerts-slack-channel >>
+          template: basic_fail_1
+  cas3_e2e_tests:
+    parallelism: 4
+    circleci_ip_ranges: true # opt-in to jobs running on a restricted set of IPs
+    docker:
+      - image: cypress/browsers:node16.14.2-slim-chrome100-ff99-edge
+    parameters:
+      environment:
+        type: string
+        default: dev
+    steps:
+      - run:
+          name: Checkout e2e repo
+          command: |
+            git clone https://github.com/ministryofjustice/hmpps-temporary-accommodation-ui.git .
+      - run:
+          name: install
+          command: apt-get install xz-utils
+      - run:
+          name: Update npm
+          command: 'npm install -g npm@latest'
+      - node/install-packages
+      - run:
+          name: E2E Check - << parameters.environment >>
+          command: |
+            TESTS=$(circleci tests glob "e2e/tests/*.feature" | circleci tests split --split-by=timings | paste -sd ',')
+            npm run test:e2e:ci --\
```

Comment:
I don't expect this to change much in the frontend but remembering to come and update this seems like a very easy thing to miss. Perhaps we could leave a link to this file in [our frontend e2e.env files](https://github.com/ministryofjustice/hmpps-temporary-accommodation-ui/blob/main/e2e.local.env) as a pointer?

(I assume it's impossible to load environment variables from another circleci context and that even if we could that might have unintended consequences anyway).

Reply:
I've added all the shared secrets to a [Context](https://app.circleci.com/settings/organization/github/ministryofjustice/contexts/4a855c71-1cc7-498a-b7bf-4e7d1cfbe1b1?return-to=%2F) here, so we can share them between projects.